projects/components/src/select/select.component.ts

```diff
@@ -114,7 +114,7 @@ import { SelectSize } from './select-size';
               *ngFor="let item of items"
               (click)="this.onSelectionChange(item)"
               class="select-option"
-              [ngClass]="this.size"
+              [ngClass]="this.getStyleClassesForSelectItem(item)"
```

Comment:
Can we avoid the polling?  Probably use htMemoize?

Reply:
@anandtiwary done.